templates/000-feature-template.md

```diff
@@ -27,7 +27,7 @@
 
 > Explain the UI changes that your proposal would need (if applicable). This could mean:
 
-> Provide a high level diagram or wireframe about the change.
+> Provide at least a low-fidelity wireframe of the change.
 > If applicable, suggest error, empty and loading states for the change.
```

Comment:
I don't think we're actually doing any RFCs with this level of specificity -- maybe to simplify, we simply remove this so that the bulleted list is gone and the requirement is clear?

Reply:
Good call, removing it.